ecs-cli/modules/commands/configure/configure_command.go

```diff
@@ -149,24 +166,16 @@ func configureFlags() []cli.Flag {
 			),
 		},
 		cli.StringFlag{
-			Name:  flags.ComposeProjectNamePrefixFlag,
-			Value: flags.ComposeProjectNamePrefixDefaultValue,
-			Usage: fmt.Sprintf(
-				"[Optional] Specifies the prefix added to an ECS task definition created from a compose file. Format <prefix><project-name>.",
-			),
-		},
-		cli.StringFlag{
 			Name:  flags.ComposeServiceNamePrefixFlag,
 			Value: flags.ComposeServiceNamePrefixDefaultValue,
```

Comment:
Should we remove this default value?

Reply:
Wait- I thought Compose Service Name Prefix was not being removed- therefore it should not be modified and the default value should remain the same.